src/saveload/saveload.h

```diff
@@ -377,6 +377,7 @@ enum SaveLoadVersion : uint16_t {
 	SLV_DEPOT_UNBUNCHING,                   ///< 331  PR#11945 Allow unbunching shared order vehicles at a depot.
 	SLV_AI_LOCAL_CONFIG,                    ///< 332  PR#12003 Config of running AI is stored inside Company.
 	SLV_SCRIPT_RANDOMIZER,                  ///< 333  PR#12063 Save script randomizers.
+	SLV_NO_NUMBER_FORMATS,                  ///< 334  PR#11993 Number format rewriting; removal of saving of number formats in the savegame.
```

Comment:
Adding or removing settings requires no savegame bump. Only data migrations do.

Reply:
Hmm... how is this going to work?

I removed the settings completely and now some save game versions fail to load (I reckon from version 118).
Or should I have placed `SLV_SCRIPT_RANDOMIZER + 1` as last version in the settings ini file? In that case I would have gotten the comment that I just should've made a saveload version.
Or should I just have placed `SLV_SCRIPT_RANDOMIZER` as last version in the settings ini file? That would like have gotten a comment about misusing the saveload version as well.

So, what am I missing here?